src/fsharp/fsi/console.fs

```diff
@@ -127,6 +134,8 @@ module internal Utils =
             | Some methInfo -> methInfo        
 
 
+#if FX_REDUCED_CONSOLE
```

Comment:
Oh there is so much of that that can be cleaned up it's fantastic.  Maarten Range is my preprocessor hero :-)